src/store/swap.ts

```diff
@@ -225,23 +226,23 @@ export const toAmountAtom = atom(
     }
 
     // Auto-fill "from" amount when "to" amount is entered.
-    if (swapDirection === SwapDirection.TO_ANCHOR) {
+    if (swapDirection === SwapDirection.TO_STABLE) {
       const fee = governedParams.GiveStableFee;
-      const newFromAmount = floorMultiplyBy(
+      const newFromAmount = ceilMultiplyBy(
```

Comment:
I don't know if this math is right. Please pull this calculation out into a utility with a test. Ideally one that would have failed before this PR.

Reply:
Done